Makefile

```diff
@@ -21,4 +21,4 @@ test:
 
 .PHONY: audit
 audit:
-	nancy go.sum
\ No newline at end of file
+	go list -json -m all | nancy sleuth
```

Comment:
I think we are ok to replace this script to the latest one 
go list -json -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore
according to the thread
https://onswebsite.slack.com/archives/CCJMKPHPW/p1614697049123400

Reply:
Ah thank you, I didn't see this comment in slack, will use this.